packages/admin/admin/src/dataGrid/excelExport/DataGridExcelExportButton.tsx

```diff
@@ -0,0 +1,44 @@
+import { Button, CircularProgress } from "@mui/material";
+import * as React from "react";
+import { FormattedMessage } from "react-intl";
+
+import { FileIcon } from "../../fileIcons/FileIcon";
+import { ExportApi } from "./useDataGridExcelExport";
+
+interface Props {
+    exportApi: ExportApi;
+    onClick?: () => void;
+    loadingComponent?: React.ReactNode;
+    onError?: () => void;
+}
+
+export const DataGridExcelExportButton: React.FunctionComponent<Props> = ({ onClick, children, exportApi, loadingComponent, onError }) => {
+    const onClickButtonPressed = () => {
+        if (onClick != null) {
```

Comment:
The type says it's either undefined or a function but you do a loose equality check for null. While this works, the cleaner way to do this is:

```suggestion
        if (onClick !== undefined) {
```

or simply

```suggestion
        if (onClick) {
```